web-server/src/content/Service/SystemLogs.tsx

```diff
@@ -0,0 +1,59 @@
+import { Box, Typography } from '@mui/material';
+import { useRouter } from 'next/router';
+import { useEffect, useRef, useMemo } from 'react';
+
+import { ServiceNames } from '@/constants/service';
+import { useSelector } from '@/store';
+
+export const SystemLogs = ({ serviceName }: { serviceName: ServiceNames }) => {
+  const router = useRouter();
+  const containerRef = useRef<HTMLDivElement>(null);
+  const services = useSelector((state) => state.service.services);
+  const logs = useMemo(() => {
+    return serviceName ? services[serviceName]?.logs || [] : [];
```

Comment:
Based on prop types serviceName isn't optional, so it will be always true.
so simply use `services[serviceName]?.logs || []`

Reply:
in that case we dont even have to redirect to `system-logs` right